src/commons/workspace/WorkspaceReducer.ts

```diff
@@ -122,6 +126,15 @@ export const WorkspaceReducer: Reducer<WorkspaceManagerState> = (
   }
 
   switch (action.type) {
+    case ADD_TOKEN_COUNT:
```

Comment:
Would `SET_TOKEN_COUNT` make more sense instead of `ADD_TOKEN_COUNT` given the reducer's behaviour?

Reply:
That's true. I'll make sure to change all references to set token count instead